packages/syft/src/syft/lib/__init__.py

```diff
@@ -257,6 +258,18 @@ def create_lib_ast(client: Optional[Any] = None) -> Globals:
 lib_ast = create_lib_ast(None)
 
 
+def register_library(lib: str) -> None:
+    """
+    Add newly imported library to shared lib space.
+    """
+    package_support_path = "__SYFT_PACKAGE_SUPPORT"
+    # load(lib, ignore_warning=True)
+    if package_support_path not in sys.modules:
+        sys.modules[package_support_path] = set()
+    sys.modules[package_support_path].add(lib)
+    # bind_library(lib)
+
+
 @wrapt.when_imported("gym")
```

Comment:
Q: You need to comment this for `syft_gym` to work. 
CC: @arpitvaghela

Reply:
Comment till syft_gym is ready and then finally remove all the related files from syft. Literally moving libs out of syft :smile: